task/02-numbers-tasks.js

```diff
@@ -202,9 +204,15 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    for (let i = 2; i <= Math.sqrt(n); i++) {
+        if (n % i == 0) {
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"